_sass/components/_hero.scss

```diff
@@ -23,15 +23,51 @@
 //
 // Styleguide components.hero
 
-.hero{
+.hero {
 	padding-top: 2em;
 	padding-bottom: 2em;
-
 }
-.hero-title_text{
-	font-size: 3rem; //
+
+.hero-title_text {
+	font-size: 3rem;
 }
-.hero-button{
+
+.hero-button {
 	margin: .5em;
 	margin-top: 1em;
 }
+
+.hero-left {
+	padding-top: $base-padding-large;
+	padding-bottom: $base-padding-large;
+}
+
+.hero-right {
+	padding-bottom: 0px;
```

Comment:
Line should be indented with spaces, not tabs<br>`0px` should be written without units as `0`